File: exam-slammr-webapp/src/test/java/uk/co/claritysoftware/exam/slammr/webapp/service/model/question/AnswerOptionTest.java
package uk.co.claritysoftware.exam.slammr.webapp.service.model.question;

import org.junit.Test;

import nl.jqno.equalsverifier.EqualsVerifier;
import uk.co.claritysoftware.exam.slammr.webapp.service.model.question.AnswerOption;

/**
 * Unit test class for {@link AnswerOption}
 */
public class AnswerOptionTest {

	@Test
	public void shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(AnswerOption.class)
				.verify();
	}
}

File: exam-slammr-webapp/src/test/kotlin/uk/co/claritysoftware/exam/slammr/webapp/service/model/question/AnswerOptionTest.kt
package uk.co.claritysoftware.exam.slammr.webapp.service.model.question

import nl.jqno.equalsverifier.EqualsVerifier
import org.junit.Test

/**
 * Unit test class for [AnswerOption]
 */
class AnswerOptionTest {

	@Test
	fun shouldHonourEqualsHashcodeContract() {
		EqualsVerifier.forClass(AnswerOption::class.java)
				.verify()
	}
}